C/src/xplaneConnect.c

```diff
@@ -735,9 +744,9 @@ int sendTEXT(XPCSocket sock, char* msg, int x, int y)
 	size_t len = 14 + msgLen;
 	memcpy(buffer + 5, &x, sizeof(int));
 	memcpy(buffer + 9, &y, sizeof(int));
-	buffer[13] = msgLen;
+	buffer[13] = (unsigned char)(msgLen & 0xff);
```

Comment:
Is the `& 0xff` necessary/useful here? As far as I know, the cast should already be truncating the value of msgLen.